samples/bluetooth/peripheral_cts_client/src/main.c

```diff
@@ -0,0 +1,356 @@
+		       p_evt->current_time.exact_time_256.day);
+	}
+
+	printk("\tMonth of year %s\n",
+	       month_of_year[p_evt->current_time.exact_time_256.month]);
+	if (p_evt->current_time.exact_time_256.year == 0) {
+		printk("\tYear          Unknown\n");
+	} else {
+		printk("\tYear          %u\n",
+		       p_evt->current_time.exact_time_256.year);
+	}
+	printk("\nTime:\n");
+	printk("\tHours     %u\n", p_evt->current_time.exact_time_256.hours);
+	printk("\tMinutes   %u\n", p_evt->current_time.exact_time_256.minutes);
+	printk("\tSeconds   %u\n", p_evt->current_time.exact_time_256.seconds);
+	printk("\tFractions %u/256 of a second\n",
+	       p_evt->current_time.exact_time_256.fractions256);
+
+	printk("\nAdjust reason:\n");
+	printk("\tDaylight savings %x\n",
+	       p_evt->current_time.adjust_reason
+		       .change_of_daylight_savings_time);
+	printk("\tTime zone        %x\n",
+	       p_evt->current_time.adjust_reason.change_of_time_zone);
+	printk("\tExternal update  %x\n",
+	       p_evt->current_time.adjust_reason.external_reference_time_update);
+	printk("\tManual update    %x\n",
+	       p_evt->current_time.adjust_reason.manual_time_update);
+}
+
+static void bt_cts_c_evt_handler(struct bt_gatt_cts_c *cts_c,
+				 bt_gatt_cts_c_evt_t *p_evt)
+{
+	switch (p_evt->evt_type) {
+	case BT_GATT_CTS_C_EVT_CURRENT_TIME:
+		printk("Current Time received.\n");
+		current_time_print(p_evt);
+		break;
+
+	case BT_GATT_CTS_C_EVT_INVALID_TIME:
+		printk("Invalid Time received.\n");
+		break;
+
+	default:
+		/* No implementation needed. */
+		break;
+	}
+}
+
+static int cts_c_init(void)
+{
+	int err;
+	struct bt_gatt_cts_c_init_param cts_c_init;
+
+	cts_c_init.evt_handler = bt_cts_c_evt_handler;
+	err = bt_gatt_cts_c_init(&cts_c, &cts_c_init);
+
+	return err;
+}
+
+static void button_changed(uint32_t button_state, uint32_t has_changed)
+{
+	uint32_t buttons = button_state & has_changed;
+	int err;
+
+	if (buttons & KEY_READ_TIME) {
+		err = bt_gatt_cts_c_current_time_read(&cts_c, K_NO_WAIT);
+		if (err) {
+			printk("Failed reading current time (err: %d)\n", err);
+		}
+	}
+}
+
+static int init_button(void)
+{
+	int err;
+
+	err = dk_buttons_init(button_changed);
+	if (err) {
+		printk("Cannot init buttons (err: %d)\n", err);
+	}
+
+	return err;
+}
+
+void main(void)
+{
+	int blink_status = 0;
+	int err;
+
+	printk("Starting Current Time Service client example\n");
+
+	err = cts_c_init();
+	if (err) {
+		printk("CTS client init failed (err %d)\n", err);
+		return;
+	}
+
+	err = dk_leds_init();
+	if (err) {
+		printk("LEDs init failed (err %d)\n", err);
+		return;
+	}
+
+	err = init_button();
+	if (err) {
+		printk("Button init failed (err %d)\n", err);
+		return;
+	}
+
+	err = bt_enable(NULL);
+	if (err) {
+		printk("BLE init failed (err %d)\n", err);
+		return;
+	}
+
+	if (IS_ENABLED(CONFIG_SETTINGS)) {
+		settings_load();
+	}
+
+	bt_conn_cb_register(&conn_callbacks);
+	if (err) {
+		printk("Failed to register connection callbacks.\n");
```

Comment:
Remove the dot from log since other `printk` calls in the sample mostly don't print dot

Reply:
Done.